src/Requests/Companies/MemberRequest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace TruckersMP\Requests\Companies;
+
+use TruckersMP\Models\CompanyMember;
+use TruckersMP\Requests\Request;
+
+class MemberRequest extends Request
+{
+    /**
+     * The ID of the requested company.
+     *
+     * @var int
+     */
+    protected $companyId;
+
+    /**
+     * The DID of the requested member.
```

Comment:
A typo in this comment.

```suggestion
     * The ID of the requested member.
```